src/services/elasticsearch.js

```diff
@@ -31,6 +32,27 @@ class ElasticSearchService {
     }
     return [];
   }
+
```

Comment:
enabled users to add a quote

I am not very familiar with this app or the API used, so code will need to be double checked for accuracy